src/bindings/python/fluxacct/accounting/job_archive_interface.py

```diff
@@ -226,7 +213,10 @@ def get_job_records(conn, bank, default_bank, **kwargs):
         # special case for unit tests in test_job_archive_interface.py
         return add_job_records(result)
 
-    job_records = add_job_records(jobs)
+    is_default_bank = bank == default_bank
+    jobs = filter_jobs_by_bank(
+        result, bank if not is_default_bank else default_bank, is_default_bank
```

Comment:
maybe `result, bank if bank != default_bank else default_bank, is_default_bank`? I found this a bit hard to read

Reply:
Actually how is this different than `filter_jobs_by_bank(result, bank)`? Unless there's some important difference between `bank == default_bank` and `bank is default_bank`?

Reply:
I think I should just break these up into separate lines; it'll make it clearer - sorry about that. To the best of my knowledge there isn't an important difference between `bank == default_bank` and `bank is default_bank`.

Reply:
I just realized I missed the third argument to `filter_jobs_by_bank()`, where you pass in `is_default_bank`. My bad. It makes more sense now.